src/main/java/uk/gov/hmcts/reform/sscs/bulkscancore/validators/CaseValidator.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.sscs.bulkscancore.validators;
+
+import java.util.Map;
+import uk.gov.hmcts.reform.sscs.bulkscancore.domain.CaseValidationResponse;
+
+/**
+ * Each jurisdiction service needs to provide implementation of this interface
+ * In case validation is not required then return original case data in the transformation.
```

Comment:
is this comment accurate?

Reply:
So there will be scenario where say some jurisdiction may want to do validation before case is created and some may just want to create case if transformation is successful. Since this will be common interface each service will need to do this. We can update the comment or if there is better implementation then we can discuss :)

Reply:
I think the comment is reasonable but looking at the current implementation in CCDCallbackHandler, it doesn't look to me that its what is being done?

Reply:
yes there is no implementation for the interface as of now but will be added by Jack later